src/components/sign_up/formFields.js

```diff
@@ -218,7 +217,7 @@ export const covidStudentFields = [
       .typeError("Please provide date in MM-DD-YYYY format")
       .test("test age minimum", "", function (value) {
         const today = DateTime.now();
-        const parsedBirthdate = DateTime.fromFormat(value, "MM'-'dd'-'yyyy");
+        const parsedBirthdate = DateTime.fromJSDate(value);
```

Comment:
isn't 'value' a string?

Reply:
Actually returned as a javascript date object. Confirmed it working on my end.